tests/testcases/core/domain/services/graphql/mutators/PriceDeleteTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace EventEspresso\tests\testcases\core\domain\services\graphql\mutators;
+
+use EventEspresso\tests\testcases\core\domain\services\graphql\GraphQLUnitTestCase;
+use GraphQLRelay\Relay;
+
+class PriceDeleteTest extends GraphQLUnitTestCase
+{
+    public $client_mutation_id;
+    public $admin;
+    public $subscriber;
+    public $entity;
+
+    public function setUp()
+    {
+        // before
+        parent::setUp();
+        if (PHP_VERSION_ID < 70000) {
+            $this->markTestSkipped(
+                'WP GraphQL compatible with PHP 7+ only'
+            );
+            return;
+        }
+
+
+        $this->client_mutation_id = 'someUniqueId';
+
+        $this->admin = $this->factory()->user->create([
+            'role' => 'administrator',
+        ]);
+
+        $this->subscriber = $this->factory()->user->create([
+            'role' => 'subscriber',
+        ]);
+
+        $this->entity = $this->new_model_obj_with_dependencies('Price');
+    }
+
+    /**
+     * This processes a mutation to delete an entity
+     *
+     * @return array
+     */
+    public function deleteMutation($id, $deletePermanently = false)
+    {
+        $mutation = '
+		mutation DeletePrice($input: DeleteEspressoPriceInput!) {
+            deleteEspressoPrice(input: $input) {
+				clientMutationId
+				espressoPrice {
+                    id
+                    dbId
+					name
+                    isTrashed
+				}
+			}
+		}
+		';
+
+        $variables = wp_json_encode([
+            'input' => [
+                'clientMutationId'  => $this->client_mutation_id,
+                'id'                => $id,
+                'deletePermanently' => $deletePermanently,
+            ]
+        ]);
+
+        return do_graphql_request($mutation, 'deleteEntity', $variables);
```

Comment:
even this logic appears to be copied a few times and could have been moved to the parent class, so then the text case classes would only have to pass their specific details to the parent method:

```php
        $mutation = '
		mutation DeletePrice($input: DeleteEspressoPriceInput!) {
            deleteEspressoPrice(input: $input) {
				clientMutationId
				espressoPrice {
                    id
                    dbId
					name
                    isTrashed
				}
			}
		}
		';
        return $this->runDeleteMutation($mutation, $id, $deletePermanently);
```